src/main/java/agh/edu/pl/MapNotes/model/User.java

```diff
@@ -0,0 +1,48 @@
+package agh.edu.pl.MapNotes.model;
+
+import javax.persistence.*;
+import java.util.HashMap;
+
+
+/**
+ * User is an entity, which contain user info.
+ * Its have its unique id.
+ * User has password and mail for authentication.
+ *
+ * @see User
+ * @see UserRepository
+ */
+
+
+@Entity
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Lob
+    private HashMap<String, Object> data;
```

Comment:
Absolutnie bym tak nie robił. W przypadku mapy robiliśmy tak aby ułatwić sobie dodawanie atrybytów, ale nie było to do końca poprawne. Tak w przypadku hasła jest to absolutnie nie poprawne. Ponad to hasło użytkownika nigdy nie powino być a) odsyłana od serwera do użytkownika, b) trzymane w postaci jawnej.